### PR TITLE
status display implemented

### LINKED ISSUE #14: Status display
Most resources have a status displayed along with it. Please add this status to the overview and color it accordingly.
See modelsv1Status in swagger API.

### PR DESCRIPTION
**Pull request by anna-brandt**
Added status display with different colors.

fixes #14 and #18 

image

image

### CODE REVIEW

File: src/app/services/api.service.ts [hunk 1]
```diff
@@ -110,7 +114,7 @@ export class ApiService {
   //Executes a http post request to add a user to a project
   addUsertoProject(user_id) {
     return new Promise(resolve => {
-      var post_object = { user_id: user_id, scope: ["READ", "WRITE"], projectId: this.project.project["id"] }
+      var post_object = { userId: user_id, scope: [ "RIGHT_UNSPECIFIED"], projectId: this.project.project["id"] }
```
Comment (MariusDieckmann) [inline src/app/services/api.service.ts:117]: I think RIGHT_UNSPECIFIED is not right here. It should probably [ "RIGHT_READ", "RIGHT_WRITE"]

**Review by MariusDieckmann — CHANGES_REQUESTED** [review]
Status changes okay, but change handling of scope when adding user.

File: src/app/services/api.service.ts (hunk 1 above)
Comment (nknoppik) [reply src/app/services/api.service.ts]: Changed in Commit 0fda10285f9c1d346588d468e8c50ec2bf3cf2df

**Review by MariusDieckmann — APPROVED** [review]
Looks good to me, lgmt.
We should at some point consider handle the rights properly, but this depends on decisions to be taken in the backend.

**Pull request merged**